english/vote-the-president/Hillary.md

```diff
@@ -0,0 +1 @@
+Congratulations. You made another clinton president again. Get ready to see some great extra marital affairs that will give a lots of money to media and spicy news for the Americans.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.